com.gigitux.gtkwhats.json

```diff
@@ -0,0 +1,39 @@
+{
+  "app-id": "com.gigitux.gtkwhats",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.36",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "gtkwhats",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=session-bus",
```

Comment:
What do you need the session-bus for?

Reply:
I need this to make notifications from WebKit work

Reply:
you need `--talk-name=org.freedesktop.Notifications` instead

Reply:
I have this error when open application:
```
Failed to register: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
```

Reply:
Your issue comes from https://github.com/gigitux/gtkwhats/blob/master/src/constants/constants.rs#L3. The app_id you pass to the gtk::Application should be the same as the one you define on your flatpak manifest